2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:254] - DEBUG: Attempting to acquire lock 140489088895440 on /home/snowwolf/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:257] - DEBUG: Lock 140489088895440 acquired on /home/snowwolf/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:286] - DEBUG: Attempting to release lock 140489088895440 on /home/snowwolf/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:289] - DEBUG: Lock 140489088895440 released on /home/snowwolf/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:254] - DEBUG: Attempting to acquire lock 140485705198992 on /home/snowwolf/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:257] - DEBUG: Lock 140485705198992 acquired on /home/snowwolf/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:286] - DEBUG: Attempting to release lock 140485705198992 on /home/snowwolf/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
2024-02-08 23:21:20,454 - /home/snowwolf/miniconda3/envs/llama2/lib/python3.11/site-packages/filelock/_api.py[line:289] - DEBUG: Lock 140485705198992 released on /home/snowwolf/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
